### ISSUE TITLE
run cellchat <- computeCommunProb(cellchat) error

### ISSUE THREAD

**Issue by woshizy** [sqjin/CellChat#506]
Please ask the teachers, in the standard process of running single cell CellChat function, cellchat <- computeCommunProb(cellchat) reports error: "Error in computeCommunProb(cellchat): object 'spot.size' not found" is What is the reason?

**Comment by lixiangyzju** [comment]
I met the same error when I updated CellChat to version1.6

**Comment by sqjin** [comment]
@woshizy @lixiangyzju Have you updated the CellChat object via `updateCellChat`

**Comment by woshizy** [comment]
I met the same error when I updated CellChat to version1.6 and 1.5

**Comment by lixiangyzju** [comment]
cellchat <- updateCellChat(cellchat)
Error in methods::new(Class = "CellChat", data.raw = object@data.raw,  : 
  object 'images' not found

**Comment by sqjin** [comment]
@lixiangyzju There should be no issue when you update your object for the first time. Anyway, I just update the code so that it can solve your error. 

@woshizy You can find the version 1.5 in the previous release.

**Comment by woshizy** [comment]
@sqjin As mentioned above, I met the same error when I run CellChat at version 1.5

**Comment by lixiangyzju** [comment]
@sqjin great thanks for your help! After reinstalled CellChat (Ver.1.6.0), I can update the cellchat object successfully. However, the spot size issue is still there. My R version is 4.1.3.

 Run CellChat on sc/snRNA-seq data <<< [2022-11-13 22:07:31]"
  Error in computeCommunProb(cellchat, trim = 0.1, LR.use = NULL, raw.use = TRUE,  : 
  object 'spot.size' not found

**Comment by sqjin** [comment]
@lixiangyzju @woshizy It is fixed now. Please re-install it and let me know if it does not work.  @woshizy CellChat v1.5 should not have this error.

**Comment by hehejunxian** [comment]
I met the same error when I run CellChat at version1.6
And i cannot update the CellChat object via updateCellChat 

cellchat <- updateCellChat(cellchat)
Error in methods::new(Class = "CellChat", data.raw = [object@data.raw](mailto:object@data.raw), :
object 'images' not found

**Comment by sqjin** [comment]
@hehejunxian Have you re-installed it today?  It should be no issue now.

**Comment by hehejunxian** [comment]
@sqjin Yes, i re-installed it today, and there was no error, thanks for your help!
